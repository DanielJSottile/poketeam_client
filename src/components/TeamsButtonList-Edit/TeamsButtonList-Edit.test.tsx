import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
//import { render } from '@testing-library/react';
import TeamsButtonListEdit from './TeamsButtonList-Edit';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<TeamsButtonListEdit/>', (): void => {
  it('renders without crashing', (): void => {
    const div: HTMLDivElement = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <TeamsButtonListEdit />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it('snapshot', (): void => {
    const wrapper = shallow(<TeamsButtonListEdit />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
