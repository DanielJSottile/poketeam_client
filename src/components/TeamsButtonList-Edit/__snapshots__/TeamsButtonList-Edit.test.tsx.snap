// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamsButtonListEdit/> snapshot 1`] = `
<Fragment>
  <section
    className="folders-list"
  >
    <h3>
      Teams:
    </h3>
    <div>
      <h3>
        Create or Click On a Folder to Get Started!
      </h3>
    </div>
    <div />
    <div />
  </section>
</Fragment>
`;
